README.md

```diff
@@ -113,20 +113,22 @@ RInAppMessaging.logEvent(CustomEvent(withName: "any_event_name_here", withCustom
 
 A preference is what will allow IAM to identify users for targeting and segmentation. At the moment, IAM will take in any of the following identifiers:
 
-1.  RakutenID
+1.  RakutenID - Any value that is considered by the app as user identifier.
 2.  UserID - The ID when registering a Rakuten account. e.g. an email address
+2.  IDTrackingIdentifier - The value provided by the internal identity SDK as the "tracking identifier" value.
 3.  AccessToken - This is the token provided by the internal RAuthentication SDK as the "accessToken" value
 
 To help IAM identify users, please set a new preference every time a user changes their login state i.e. when they log in or log out.  
 After logout is complete please call  `registerPreference()` with nil parameter.  
 Not all identifiers have to be provided.  
-**NOTE:** For our internal users - for user targeting you must provide an accessToken. If you are setting an accessToken you must also provide associated userId in `IAMPreference`.
+**NOTE:** For our internal users - to enable user targeting when you are using the internal RAuthentication SDK, you must provide an accessToken along with associated userId in `IAMPreferenceBuilder`.
 
 ```swift
 let preference = IAMPreferenceBuilder()
     .setUserId("testaccount@gmail.com")
     .setRakutenId("testaccount")
     .setAccessToken("27364827346")
+    .setIDTrackingIdentifier("998236")
```

Comment:
>accessToken and idTrackingIdentifier shouldn't be used at the same time

I get this is just an example but there's an assertion that says you shouldn't use both of these at the same time 😅

Reply:
Yeah I was wondering about that. The limitation is just for Rakuten devs and I thought that violating that assertion will be difficult for them in normal environment (but who knows...). Maybe I just leave that example as it was?

Reply:
Or could be commented out too to show that it's an option

Reply:
Good idea